src/file_path.h

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef  _WIN32
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers
+#endif
+
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+
+// Windows Header Files:
+#include <windows.h>
+#define ENABLE_INTSAFE_SIGNED_FUNCTIONS // Only unsigned intsafe math/casts available without this def
+#include <intsafe.h>
+#include <tchar.h>
+
+using path_type = std::wstring;
+
+[[nodiscard]] inline std::wstring ConvertToW(const UINT codePage, const std::string_view source) {
```

Comment:
Copied from https://github.com/microsoft/terminal/blob/3486111722296f287158e0340789c607642c1067/src/types/convert.cpp#L20  . Dropped all error handling because we don't have WIL included.